### PR TITLE
Add test coverage for `aea/cli/package_utils`

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

Adds test coverage for `aea/cli/package_utils`

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/test_cli/test_utils/test_package_utils.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+        ):
+            assert (
+                cast(
+                    PublicId,
+                    is_item_with_hash_present(
+                        str(self.t / agent_name), agent_config, "Qm"
+                    ),
+                ).name
+                == "signing"
+            )
```
Comment (Karrenbelt) [inline tests/test_cli/test_utils/test_package_utils.py:207]: why cast to `PublicId` here but not above?

Comment (angrybayblade) [reply tests/test_cli/test_utils/test_package_utils.py:205]: Not sure exactly why, but the mypy only complained about this one

**Comment by angrybayblade** [comment]
The ubuntu failures are flaky: https://github.com/valory-xyz/open-aea/issues/408

File: aea/cli/utils/package_utils.py [hunk 2]
```diff
@@ -104,7 +104,7 @@ def verify_private_keys_ctx(
         ).agent_config
         if ctx is not None:
             ctx.agent_config = agent_config
-    except ValueError as e:  # pragma: nocover
+    except ValueError as e:
```
Comment (Karrenbelt) [inline aea/cli/utils/package_utils.py:107]: :+1:

**Review by Karrenbelt — DISMISSED** [review]

**Comment by Karrenbelt** [comment]
failures seen:
- ubuntu 3.8, 3.9, 3.10
   ```
   FAILED tests/test_helpers/test_exec_timeout.py::test_supervisor_not_started
   ```
   - #408 
- windows 3.7
   ```
   FAILED tests/test_test_tools/test_test_contract.py::TestBaseContractTestCaseSetup::test_contract_setup_transaction_receipt_not_found
    FAILED tests/test_protocols/test_generator/test_generator.py::ProtocolGeneratorTestCase::test_generate_full_mode_protolint_error
    FAILED tests/test_cli/test_build.py::TestAEABuildMainEntrypoint::test_build
    FAILED tests/test_cli/test_build.py::TestAEABuildPackageEntrypoint::test_build
   ```

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**